Test/DatabaseTests/DatabaseLoaderTests.cs

```diff
@@ -107,7 +107,7 @@ public void FilesLoading()
             Loaders.LoadElements(Path.Combine(TestContext.CurrentContext.TestDirectory, "elements2.dat"));
 
             var unimodMods = Loaders.LoadUnimod(Path.Combine(TestContext.CurrentContext.TestDirectory, "unimod_tables2.xml")).ToList();
-            Assert.AreEqual(2639, unimodMods.Count); // UniMod PTM list may be updated at some point, causing the unit test to fail
+            Assert.AreEqual(2633, unimodMods.Count); // UniMod PTM list may be updated at some point, causing the unit test to fail
```

Comment:
this will probably fail on appveyor because it re-downloads the latest version of unimod mods and you probably have an older version of the mods list on your machine (same with below unit tests)

Reply:
try deleting mzLib\Test\bin\x64\Debug on your machine and see if the unit tests pass